aws/v11.4.1/README.md

```diff
@@ -0,0 +1,80 @@
+# :zap: Giant Swarm Release v11.4.1 for AWS :zap:
+
+This release re-activates the recent AWS [release of high-availability (HA) masters](https://github.com/giantswarm/releases/tree/master/aws/v11.4.0) while fixing issues with certain OIDC configurations.
+
```

Comment:
```suggestion
**Note for Solution Engineers:** This release contains an external-dns fix introduced in
[11.3.2](https://github.com/giantswarm/releases/blob/master/aws/v11.3.2/release-notes.md).
It requires manual intervention for cluster upgrades in China to work. When upgrading a
cluster, existing ingress `A+TXT` record sets do not get replaced with `CNAME+TXT` record sets
even when external-dns is configured with CNAMEs as preferred. After upgrading, delete the
ingress `A+TXT` record sets. external-dns will then automatically create `CNAME+TXT` record
sets.
```

@sslavic , I know you hate it. But I think we still need this note to persist here? :P Please confirm.

Reply:
I just put it in and merge here - okay? 
I kind of want to be done with this :sweat_smile: